forecasting/src/autogluon/forecasting/models/gluonts_model/mqcnn/default_parameters.py

```diff
@@ -0,0 +1,11 @@
+def get_default_parameters():
```

Comment:
can we instead name this file `parameters.py` like in TabularPrediction? It should also be stored in a subdirectory `hyperparameters/` inside of the `mqcnn/` directory, just as in TabularPrediction.